heka/_service.sls

```diff
@@ -0,0 +1,269 @@
+
+{%- macro load_grains_file(grains_fragment_file) %}{% include grains_fragment_file %}{% endmacro %}
```

Comment:
Since not all services have meta/heka.yml until you add it to all formulas, this macro will fail.
Thanks to jinja2 for ignoring missing:

{%- macro load_grains_file() %}{% include grains_fragment_file ignore missing %}{% endmacro %}